metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -0,0 +1,138 @@
+import json
+import os
+import uuid
+from functools import wraps
+from pathlib import Path
+from typing import Any, Callable, Dict, Optional, TypeVar, Union
+
+import requests
+
+import datahub as datahub_package
+
+GA_VERSION = 1
+GA_TID = "UA-214428525-1"
+
+LOCAL_DIR = Path(__file__).resolve().parent
+
+DATAHUB_FOLDER = Path(os.path.expanduser("~/.datahub"))
```

Comment:
This is present in https://github.com/linkedin/datahub/blob/master/metadata-ingestion/src/datahub/cli/cli_utils.py. I think we should use that

Reply:
I think that one's slightly different – it's a single file (`. datahubenv`) rather than a folder. We already use this folder in our Gradle and Docker scripts. I didn't want to tack on to this file for now because it would be a bit messier.

Reply:
yes you are right. I missed that diff.